system/emergency_handler/include/operators/emergency_stop.hpp

```diff
@@ -0,0 +1,67 @@
+// Copyright 2024 TIER IV, Inc.
```

Comment:
Place private headers that are not used by other packages under `src` directory.
https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/directory-structure/#include